client/my-sites/plugins-wpcom/info-header.jsx

```diff
@@ -5,7 +5,7 @@ import Card from 'components/card';
 export const InfoHeader = React.createClass( {
 	render() {
 		return (
-			<Card>
+			<Card className="wpcom-plugins-info-header">
```

Comment:
In this case, with `info-header` being a sub-component of `plugins-wpcom`, the class should be more like `.plugins-wpcom__info-header`. Any class attribute within the `plugins-wpcom` folder should have the folder as the prefix.